src/gpodder/gtkui/model.py

```diff
@@ -539,18 +541,23 @@ def __init__(self, cover_downloader):
         self._cover_downloader = cover_downloader
 
         self.ICON_DISABLED = 'gtk-media-pause'
+        self.ICON_ERROR = 'dialog-warning'
 
     def _filter_visible_func(self, model, iter, misc):
+        channel = model.get_value(iter, self.C_CHANNEL)
+
         # If searching is active, set visibility based on search text
         if self._search_term is not None:
-            if model.get_value(iter, self.C_CHANNEL) == SectionMarker:
+            if channel == SectionMarker:
                 return True
             key = self._search_term.lower()
             columns = (model.get_value(iter, c) for c in self.SEARCH_COLUMNS)
             return any((key in c.lower() for c in columns if c is not None))
 
         if model.get_value(iter, self.C_SEPARATOR):
             return True
+        elif channel.__class__ == GPodcast and channel._update_error is not None:
```

Comment:
Are you checking for the class here to discard the 'All Episodes from all podcasts' item?

Reply:
The proxy channel class (all episodes) has an update_errors field to prevent it from failing for everything that uses description. The class check there is to exclude the category labels when forcing channels with errors to always show.

Reply:
ok, thanks for the clarification.

Reply:
I forgot to mention that it can be changed to use the same getattr as your icon patch, and avoid the class comparison.

```
+       elif getattr(channel, '_update_error', None) is not None:
```